.changeset/red-lemons-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'graphql-yoga': minor
+---
+
+Export YogaSchemaDefinition
```

Comment:
Export YogaSchemaDefinition and mergeSchemas

Reply:
Haha done 😅